### DISCUSSION TITLE
ik kom niet verder

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#7095] [Forum]
als ik bijvoorbeeld Breanna aan mijn eigen maten wil aanpassen dan krijg ik de melding dat alle benodigde maten voor dit patroon aanwezig zijn, maar als ik dan op het icoontje onder het meetlint klik: het flesje, dan loopt het programma vast 

![Uploaded image](https://imagedelivery.net/ouSuR9yY1bHt-fuAokSA5Q/support-1726501616056-2/public)

[![User 93399](https://backend3.freesewing.org/users/93399/card)](https://freesewing.org/users/user?id=93399)

**Comment by BenJamesBen** [comment]
_(Google Translate translation)_
Het spijt ons voor het probleem dat u hebt ondervonden. Neem eerst uw metingen opnieuw om er zeker van te zijn dat ze nauwkeurig zijn. Om er zeker van te zijn dat u de metingen correct uitvoert, volgt u de instructies op https://freesewing.org/docs/measurements . Als het Breanna-patroon nog steeds crasht wanneer u de nieuwe metingen gebruikt, laat het ons dan weten en we zullen het probleem verder onderzoeken. (We moeten mogelijk naar de metingen kijken die u gebruikt.)

_(Original English language)_
We are sorry for the issue you experienced. First, please retake your measurements to make sure they are accurate. To make sure you are taking the measurements correctly, please follow the instructions at https://freesewing.org/docs/measurements . If the Breanna pattern still crashes when you use the new measurements, please let us know and we will investigate the issue further. (We might need to look at the measurements you are using.)
